en/04_Changelogs/5.0.0.md

```diff
@@ -967,20 +967,20 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
 - Removed deprecated method `SilverStripe\Control\HTTP::get_cache_age()` without equivalent functionality to replace it
 - Removed deprecated method `SilverStripe\Control\HTTP::gmt_date()` - use native php function [`gmdate()`](https://www.php.net/manual/en/function.gmdate.php) instead
-- Removed deprecated method `SilverStripe\Control\HTTP::register_etag()` - use `ChangeDetectionMiddleware` instead
-- Removed deprecated method `SilverStripe\Control\HTTP::register_modification_date()` - use `HTTPCacheControlMiddleware::registerModificationDate()` instead
-- Removed deprecated method `SilverStripe\Control\HTTP::register_modification_timestamp()` - use `HTTPCacheControlMiddleware::registerModificationDate()` instead
-- Removed deprecated method `SilverStripe\Control\HTTP::set_cache_age()` - use `HTTPCacheControlMiddleware::singleton()->setMaxAge($age)` instead
+- Removed deprecated method `SilverStripe\Control\HTTP::register_etag()` - use [`SilverStripe\Control\Middleware\ChangeDetectionMiddleware`](api:SilverStripe\Control\Middleware\ChangeDetectionMiddleware) instead
+- Removed deprecated method `SilverStripe\Control\HTTP::register_modification_date()` - use [`SilverStripe\Control\Middleware\HTTPCacheControlMiddleware::registerModificationDate()`](api:SilverStripe\Control\Middleware\HTTPCacheControlMiddleware::registerModificationDate()) instead
+- Removed deprecated method `SilverStripe\Control\HTTP::register_modification_timestamp()` - use [`SilverStripe\Control\Middleware\HTTPCacheControlMiddleware::registerModificationDate()`](api:SilverStripe\Control\Middleware\HTTPCacheControlMiddleware::registerModificationDate()) instead
+- Removed deprecated method `SilverStripe\Control\HTTP::set_cache_age()` - use `SilverStripe\Control\Middleware\HTTPCacheControlMiddleware::singleton()->setMaxAge($age)` instead
 - Removed deprecated method `SilverStripe\Control\HTTPRequest::detect_method()` without equivalent functionality to replace it
 - Removed deprecated method `SilverStripe\Control\SimpleResourceURLGenerator::resolveUnsecuredResource()` without equivalent functionality to replace it
 - Removed deprecated method `SilverStripe\Core\BaseKernel::getIgnoredCIConfigs()` without equivalent functionality to replace it
 - Removed deprecated method `SilverStripe\Core\BaseKernel::sessionEnvironment()` without equivalent functionality to replace it
-- Removed deprecated method `SilverStripe\Core\ClassInfo::baseDataClass()` - use `DataObject::getSchema()->baseDataClass()`
-- Removed deprecated method `SilverStripe\Core\ClassInfo::table_for_object_field()` - use `DataObject::getSchema()->tableForField()`
+- Removed deprecated method `SilverStripe\Core\ClassInfo::baseDataClass()` - use `SilverStripe\ORM\DataObject::getSchema()->baseDataClass()`
+- Removed deprecated method `SilverStripe\Core\ClassInfo::table_for_object_field()` - use `SilverStripe\ORM\DataObject::getSchema()->tableForField()`
```

Comment:
```suggestion
- Removed deprecated method `SilverStripe\Core\ClassInfo::baseDataClass()` - use [`SilverStripe\ORM\DataObjectSchema::baseDataClass()`](api:SilverStripe\ORM\DataObjectSchema::baseDataClass())
- Removed deprecated method `SilverStripe\Core\ClassInfo::table_for_object_field()` - use [`SilverStripe\ORM\DataObjectSchema::tableForField()`](api:SilverStripe\ORM\DataObjectSchema::tableForField())
```

Reply:
Done.
Probably, we should update this deprecated comments in the code, that our api has the same messages.

Reply:
I don't think it's a big deal, but you can raise a card for it if you like.